src/main/java/frc/robot/subsystems/drive/OdometryThreadRunner.java

```diff
@@ -215,11 +215,17 @@ public DoubleCircularBuffer registerSignal(
             signalReadWriteLock.writeLock().lock();
             final String deviceNetwork = device.getNetwork();
             if (!CANBus.isNetworkFD(deviceNetwork)) {
-                throw new RuntimeException(String.format(
+                DriverStation.reportWarning(String.format(
```

Comment:
Did you email them about this or was that about something else?

Reply:
That was something else